### PR TITLE
134 meta2 jen5

### PR DESCRIPTION
**Pull request by jenlampton**
Another PR for https://github.com/backdrop/backdrop-issues/issues/134

### CODE REVIEW

File: core/modules/image/image.test [hunk 1]
```diff
@@ -1601,3 +1601,64 @@ class ImageStyleFlushTest extends ImageFieldTestCase {
     $this->assertEqual($this->getImageCount($style, 'private'), 0, format_string('Image style %style flushed correctly for %wrapper wrapper.', array('%style' => $style['name'], '%wrapper' => 'private')));
   }
 }
+
+/**
+ * Tests image theme functions.
+ */
+class ImageThemeFunctionWebTestCase extends DrupalWebTestCase {
+
+  public static function getInfo() {
```
Comment (quicksketch) [inline core/modules/image/image.test]: This needs to be converted to an entry in image.tests.info.

**Comment by quicksketch** [comment]
Everything looks great here except that the new test needs to have its getInfo() method replaced.

**Comment by jenlampton** [comment]
test fixed and branch pushed back up.  We'll see how this goes.

**Comment by jenlampton** [comment]
one more time.

File: core/modules/book/book.pages.inc [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
 <?php
```
Comment (quicksketch) [inline core/modules/book/book.pages.inc:2]: This added blank line at the top of this PHP file may cause headers to be sent unexpectedly. It should be removed.

**Comment by quicksketch** [comment]
Okay, did a fresh review and everything is good except somehow a blank line was added to the top of book.pages.inc. Looks like it's part of fa1b85eba363189b9385aebb412512c5b55288cc.

**Comment by quicksketch** [comment]
I manually merged this to remove the line at the top of book.pages.inc. I had to rebase the branch, so the commits changed, but the last one (fixing book) is at b6bf3c2e33f4d1a9895027a6a0e59a06311a8e98.

**Pull request closed without merging**